### ISSUE TITLE
2 things. How can I disable the economy in a certain world? and a bug report

### ISSUE THREAD

**Issue by chipamogli00** [Rocologo/BagOfGold#112]
1 )I need to know if I can disable the currency, or just use abstract currency, without the items as such. The need for this feature is that while we are playing the hunger games or any game, we have the coins in inventory and they only take away usable space. Maybe there is already a function implemented but I can't find it :(

2) I need to report a bug that has already crashed the server 2 times. When I'm in creative mode it automatically gives me a lot of money (more and more everytime) and when I try to toss it or toss it in quantity, the items on the ground start jumping until various entities are spawned. so high that my game fps lags and the whole server hangs until it finally crashes. There are times that I manage to fix it because when I see before the entities start multiplying, I just use the plugin to clean the entities on the ground and it solves, but if I don't reach it, I just eat more and more fps XD.

I have not dedicated myself to exploring the plugin 100% and maybe the answer can be very simple as there may already be an existing solution, however, I think it is better to notify it in case you do not have a common solution yet.

In conclusion, I only need an option to disable the currency item while in creative mode and disable it in certain worlds, so it will be more difficult for the bug I mentioned to cause problems, although it will always be a very good alternative to fix it from the root

This is a very nice plugin, and I really like it a lot, I am waiting for the custom currency entity that I already saw that you guys are going to add it. I laughed a lot customizing it and I sincerely hope you can add more features.
Sorry for my English, I'm from Chile and I had to rely on the translator, I hope the whole message can be understood
Regards
